Task_10/truck_delivery_rayus007/db_connector_redis.py

```diff
@@ -0,0 +1,47 @@
+import redis
+import json
+from truck_delivery_rayus007.db_connector import DBConnector
+
+
+class DbConnectorRedis(DBConnector):
```

Comment:
For implementing this class as a singleton class, please take a look at this https://python-patterns.guide/gang-of-four/singleton/#a-more-pythonic-implementation